### ISSUE TITLE
[Suggestion] A third output for refractory sawmills

### ISSUE THREAD

**Issue by Xorbah** [codetaylor/pyrotech#216]
Title. The ability to get a third output from refractory sawmills, (or any if you'd like, configurable) for pack makers to get more use out players upgrading their sawmills other than the obvious reasons. 

This would allow for people to get wood, wood chips and bark simultaneously for instance.

**Comment by nephatrine** [comment]
Yes, would like a tertiary output for bark. On a related note, ability to specify the secondary output as an IItemStack rather than just an amount of woodchips would be nice in case a modpack wants to use some other sawdust/woodchip or have some other secondary output from non-wood items.

**Comment by jonthemoron** [comment]
thank christ we're not insane and alone
